fv3core/.jenkins/actions/run_regression_tests.sh

```diff
@@ -12,8 +12,16 @@ make get_test_data
 
 if [ ${python_env} == "virtualenv" ]; then
     export TEST_ARGS="${TEST_ARGS} --junitxml=${JENKINS_DIR}/${XML_REPORT}"
-    CONTAINER_CMD="srun" make tests savepoint_tests
+    if [[ ${FV3_DACEMODE} == "True" ]]; then
+        CONTAINER_CMD="srun" make savepoint_tests
+    else
+        CONTAINER_CMD="srun" make tests savepoint_tests
```

Comment:
Should we move `make tests` onto its own action, or even delete it entirely to run on CircleCI only? I'm a little frustrated when I can't get jenkins savepoint results because the main tests failed. Maybe a question for a team meeting.